ckan/tests/lib/test_navl.py

```diff
@@ -0,0 +1,30 @@
+import nose
+import pylons
+
+from ckan.tests import helpers
+from ckan.config import environment
+
+eq_ = nose.tools.eq_
+
+
+class TestFormencdoeLanguage(object):
+    @helpers.change_config('ckan.locale_default', 'de')
+    def test_formencode_uses_locale_default(self):
+        environment.update_config()
```

Comment:
I dislike that I need to call `environment.update_config()` here in order for ckan to see that I changed the default language in the decorator. Is there another way to do this?